### DISCUSSION TITLE
Is It Possible to Add "Exploit Available" Column to Vulnerability Details Report?

### DISCUSSION THREAD

**Discussion by CodyFLee** [LimberDuck/nessus-file-analyzer#12] [Ideas]
Hello!

I love the Nessus File Analyzer, so thank you so much for sharing and maintaining.

Is it possible to add an "Exploit Available" column to the Vulnerability Details report?

So far, I have been asked if there are active exploits for the report that I just delivered, and now I find myself having to go CVE by CVE to find out.

Having this additional field would be very beneficial.

Thanks~!

**Comment by damian-krawczyk** [comment]
Hi,
Thank you for the Feedback! It should be possible 🤔. Sorry for late response. I didn't get any notification. For the future please remember to mark me in the post -> @damian-krawczyk that way I should get at least an email 🤞🏻

**Reply by CodyFLee** [reply]
Awesome! Thanks Damian. I will make sure to @damian-krawczyk in the future!

**Reply by damian-krawczyk** [reply]
@CodyFLee [done](https://github.com/LimberDuck/nessus-file-analyzer/releases/tag/v0.6.0) with extra few columns :)

----

@GitHub why I'm not getting notifications even if I'm marked in the post?
